views/patterns/02-organisms/global/footer.blade.php

```diff
@@ -65,5 +65,8 @@
   </div> <!-- /.c-footer--inner -->
 </footer> <!-- /.c-footer -->
 
-<script src="https://code.jquery.com/jquery-2.2.4.min.js"></script>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
```

Comment:
Please change this line to the following: 
`<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>`